test.js

```diff
@@ -22,7 +22,7 @@ test('compress() buffer', function * (t) {
 });
 
 test('compress() bad input', function * (t) {
-  t.throws(compress(123), 'input must be a String or a Buffer');
+  yield t.throws(compress(123), 'input must be a String or a Buffer');
```

Comment:
Without these `yield` changes, we get `Test finished, but an assertion is still pending`

Reply:
Do they even have to be generator functions?